.github/dependabot.yml

```diff
@@ -11,4 +11,12 @@ updates:
     interval: "weekly"
   open-pull-requests-limit: 10
   vulnerability-alerts:
-    enabled: true
\ No newline at end of file
+    enabled: true
+  labels:
+    - "ok-to-test"
+    - "dependencies"
```

Comment:
the labels is missing `lgtm` label.
@natasha41575 when we discussed this your suggestion was instead of `lgtm` auto add `approved` label. So should we go with `approved` instead. This will make it easier for anyone to approve dependabot PRs. I would rather go for `approved` label.